ui/scss/component/_channel-indicator.scss

```diff
@@ -1,4 +1,12 @@
 
+.channel-name {
+  width: calc(var(--card-small-width) * 2 / 3);
```

Comment:
Is this needed? I think it should be fine with just the `nowrap` and `ellipsis`. If it's extending to the edge, we should add a padding to the card.

Reply:
Without specifying the width, long text will push the price indicator to the next line. I suppose we could simplify this using a percentage width instead of the calculation. 60% or 70% should be fine.

Reply:
Gotcha. This should be fine for now, we will be moving to the new design sometime soon.

Reply:
(Proper solution is probably a flex layout rather than `calc`, but not demanding propriety. Especially since we're likely to redesign soon anyway.)